src/check_release.rs

```diff
@@ -303,19 +294,27 @@ pub(super) fn run_check_release(
         }
     }
 
-    if !results_with_errors.is_empty() {
+    let produced_errors = !results_with_errors.is_empty();
+    let produced_warnings = !results_with_warnings.is_empty();
+    if produced_errors || produced_warnings {
+        let status_color = if produced_errors {
+            AnsiColor::Red
+        } else {
+            AnsiColor::Yellow
+        };
         config
             .shell_print(
                 "Checked",
                 format_args!(
-                    "[{:>8.3}s] {} checks; {} passed, {} failed, {} unnecessary",
+                    "[{:>8.3}s] {} checks; {} passed, {} failed, {} warnings, {} skipped",
```

Comment:
`{} warnings` seems a bit inconsistent here, because this is describing how many lints generated at least one warning, not how many warnings were generated.

`warned` looks more consistent but sounds a bit weird too.

What do you think about tweaking the whole thing like:
```suggestion
"[{:>8.3}s] {} checks; {} pass, {} fail, {} warn, {} skip",
```

I'm on the fence about it personally, but leaning toward it.

Reply:
I like it because it saves characters, but it does feel a little weird not being in past tense.  It is definitely still clear and understandable, though, so I'm definitely open to changing it to this.